clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -917,7 +917,10 @@ private void handleApiVersionsResponse(List<ClientResponse> responses,
         NodeApiVersions nodeVersionInfo = new NodeApiVersions(apiVersionsResponse.data.apiKeys());
         apiVersions.update(node, nodeVersionInfo);
         this.connectionStates.ready(node);
-        log.debug("Recorded API versions for node {}: {}", node, nodeVersionInfo);
+        log.debug("Node {} has finalized features epoch: {}, finalized features: {}, supported features: {}, API versions: {}.",
+                node,
```

Comment:
It is just a style issue. Could we merge line#921 and line#922?

Reply:
@chia7712 yeah, sorry about that, now done.